### PR TITLE
Fix bug with Elasticsearch indexing unnecessarily during QR+R.

### PR DESCRIPTION
**Pull request by MikeyJC**
Update config defaults to include allowed_preview value.

## Description

Removes Elasticsearch references and indexing during QR+R, adds ```allowed_preview``` to default config function.

## Motivation and Context

Improves performance of QR+R by avoiding Elasticsearch indexing when unnecessary. Including ```allowed_preview``` in config defaults allows Elasticsearch indexing to complete without errors when indexing the Documents module.

## How To Test This

1. Run Quick Repair & Rebuild to confirm performance.
2. Manually run Elasticsearch indexing in terminal to return and view errors. Documents should now complete without error.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

**Review by jack7anderson7 — APPROVED** [review]

**Review by jack7anderson7 — DISMISSED** [review]

File: include/utils.php [hunk 1]
```diff
@@ -492,6 +492,13 @@ function get_sugar_config_defaults(): array
             'jpg'
         ],
         'upload_maxsize' => 30000000,
+        'allowed_preview' => [
+            'pdf',
+            'gif',
+            'png',
+            'jpeg',
+            'jpg'
+        ],
```
Comment (clemente-raposo) [inline include/utils.php:501]: @MikeyJC is this related with the current fix?

**Comment by clemente-raposo** [comment]
@jack7anderson7 commit needs issue number

File: include/utils.php (hunk 1 above)
Comment (clemente-raposo) [reply include/utils.php:501]: @jack7anderson7 pointed out what was on the issue description

> Including allowed_preview in config defaults allows Elasticsearch indexing to complete without errors when indexing the Documents module.

**Review by clemente-raposo — APPROVED** [review]
LGTM

**Review by jack7anderson7 — APPROVED** [review]
